src/field_theory/adjoin.lean

```diff
@@ -6,6 +6,8 @@ Authors: Thomas Browning and Patrick Lutz
 
 import field_theory.tower
 import field_theory.intermediate_field
+import field_theory.splitting_field
```

Comment:
Please make the changes in this file in a separate PR.

Reply:
#4831 which depends on #4830 which depends on #4829